bundles/core/org.eclipse.smarthome.core.test/src/test/java/org/eclipse/smarthome/core/library/types/ArithmeticGroupFunctionTest.java

```diff
@@ -201,6 +203,44 @@ public void testSumFunction() {
         assertEquals(new DecimalType("234.95"), state);
     }
 
+    @Test
+    public void testAvgFunction_dimmers() {
+        DimmerItem dimmer1 = new DimmerItem("TestDimmer1");
+        dimmer1.setState(new PercentType(100));
+        DimmerItem dimmer2 = new DimmerItem("TestDimmer2");
+        dimmer2.setState(new PercentType(0));
+
+        items.add(dimmer1);
+        items.add(dimmer2);
+
+        function = new ArithmeticGroupFunction.Avg();
+        State decimalState = function.getStateAs(items, DecimalType.class);
+
+        assertEquals(new DecimalType("50"), decimalState);
```

Comment:
>  I could leave the Arithmetic functions alone and instead do a check in GroupItem to convert a DecimalType to PercentType if the base item supports it. 

Yes, I think this would be the way to go - if the base item uses percents, the Group item should do the same (and in #1476 the Group is indeed defined as a Dimmer).